conf/application.conf

```diff
@@ -130,7 +130,9 @@ self-assessment-enrolments = "enr3,enr4"
 company-auth.host = "http://localhost:9025"
 contact-frontend.host = "http://localhost:9250"
 tax-account-router.host = "http://localhost:9280"
-business-enrolments = "enr1,enr2"
+business-enrolments = "enr1, enr2, IR-SA"
+
+sensitive-enrolments = "HMCE-VATDEC-ORG, HMCE-VATRSL-ORG, HMCE-VATVAR-ORG, HMRC-PAYERS-ORG, HMRC-VAT-GIANT, IR-CT, IR-PAYE, IR-SA-PART-ORG, IR-SA-TRUST-ORG, IR-SA"
```

Comment:
Similarly, would prefer if the project just had dummy values, and app-conf-base etc has the actual values. I worry, if it's not obvious, new readers to this will assume these values are correct and authoritative.